docs/source/links/rest.md

```diff
@@ -0,0 +1,148 @@
+---
+title: apollo-link-rest
+description: Call your REST APIs inside your graphQL queries.
+---
+
+[**Read the announcement post! ( to be pusblished soon )
+🎉**](https://dev-blog.apollodata.com/)
+
+If you have a REST API, you can now query your endpoints using GraphQL and have all your data managed by Apollo Client. There is no need to write a wrapper around your REST API.
+
+<h2 id="start">Quick start</h2>
+
+To get started, install `apollo-link-rest` from npm:
+
+```bash
+npm install apollo-link-rest --save
+```
+
+The rest of the instructions assume that you have already [set up Apollo
+Client](/docs/react/basics/setup.html#installation) in your application. After
+you install the package, you can create your rest link.
+
+```js
+const link = new RestLink({ uri: "https://api.example.com" });
+```
+
+<h2 id="options">Options</h2>
+
+REST Link takes an object with some options on it to customize the behavior of the link. The options you can pass are outlined below:
+
+- `uri`: the URI key is a string endpoint (optional when `endpoints` provides a default)
+- `endpoints`: root endpoint (uri) to apply paths to or a map of endpoints
+- `customFetch`: a custom `fetch` to handle REST calls
+- `headers`: an object representing values to be sent as headers on the request
+- `credentials`: a string representing the credentials policy you want for the fetch call
+- `fieldNameNormalizer`: function that takes the response field name and converts it into a GraphQL compliant name
+
```

Comment:
Below, you should show an example with all of the setup fields, especially providing a map of endpoints.